ocsp_status.go

```diff
@@ -107,59 +130,74 @@ func main() {
 	// TODO: extract flag descriptions into constants?
 	inresp := flag.Bool("inresp", false, "Whether to read in an OCSP responses or not")
 	incert := flag.Bool("incert", false, "Whether to read in certificate files or not")
-	ocspurl := flag.String("ocspurl", "", "User provided OCSP url, default fetch from certificate")
+	issuerFile := flag.String("issuercert", "", "Space separated list of issuing certificate files")
+	ocspurl := flag.String("ocspurl", "", "Space separated list of OCSP urls to send requests to, default fetch from certificate")
```

Comment:
Since you have more than one list value type flag, maybe you want something like this:
https://stackoverflow.com/questions/28322997/how-to-get-a-list-of-values-into-a-flag-in-golang

Reply:
So what I do right now aligns with the second answer in the stackoverflow thread you linked. It's slightly more inconvenient for the user since they need to wrap their flag in quotation marks, i.e. `-ocspurl="url1 url2 url3" instead of `-ocspurl= url1 url2 url3`.

I don't quite understand what is happening in the first answer, but if you think it is a cleaner approach, I could do it.

Reply:
SGTM no change needed here